docs/reference/primitives/app/resource/memcached.md

```diff
@@ -8,23 +8,88 @@ url: /reference/primitives/app/resource/memcached
 
 ## Definition
 
-A Memcached Resource is defined in [`convox.yml`](/configuration/convox-yml) and linked to one or more [Services](/reference/primitives/app/service).
-```html
+A Memcached Resource is defined in [```convox.yml```](/configuration/convox-yml) and linked to one or more [Services](/reference/primitives/app/service).
+
+```
+resources:
+  main:
+    type: memcached
+services:
+  web:
     resources:
-      main:
-        type: memcached
-    services:
-      web:
-        resources:
-          - main
+      - main
 ```
-## Options
 
-A Memcached Resource can have the following options configured for it (default values are shown):
-```html
+
+### AWS ElastiCache Managed Memcached Resources
+
+In addition to containerized resources, Convox v3 allows the creation of Memcached resources via AWS ElastiCache. This provides enhanced durability and managed service benefits. Below is a general example of how to define AWS ElastiCache Memcached resources:
+
+```
+resources:
+  cache:
+    type: elasticache-memcached
+    options:
+      class: cache.t3.micro
+      version: 1.6.6
+      deletionProtection: true
+      durable: true
+      encrypted: true
+      autoMinorVersionUpgrade: true
+      nodes: 2
+services:
+  web:
     resources:
-      main:
-        type: memcached
-        options:
-          version: 1.4.34
-```
\ No newline at end of file
+      - cache
+```
+
+> **Note:** The `nodes` option is required for Memcached and must be set to the desired number of nodes for the Memcached cluster.
+
+### Features
+
+- **Scalable Memcached Instances**: AWS ElastiCache Memcached supports multiple nodes for improved scalability, making it an ideal caching solution for distributed applications.
+- **Import Existing ElastiCache Memcached Instance**: You can import an existing AWS ElastiCache Memcached instance into a Convox rack for management or access via linking.
+
+### Configuration Options
+
+Below is a chart of configuration options available for AWS ElastiCache Memcached resources:
+
+| Attribute                | Type    | Description                                                                                       |
+| ------------------------ | ------- | ------------------------------------------------------------------------------------------------- |
+| **nodes**                | int     | **Required.** The number of nodes in the Memcached cluster.                                        |
+| **class**                | string  | The compute and memory capacity of the cache instance.                                             |
+| **version**              | string  | The version of the cache engine.                                                                   |
+| **deletionProtection**   | boolean | Whether to enable deletion protection for the cache instance.                                      |
```

Comment:
Aws doesn't have `deletionProtection` on their side, but this is for convox side, just incase user unintentionally remove it from convox.yml but didn't meant to delete it

Reply:
updated